examples/js-cloud-server/cloudflare-worker/tsconfig.app.json

```diff
@@ -0,0 +1,25 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+        "outDir": "../../../dist/out-tsc",
+
+        "target": "es2021" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
+        "lib": ["es2021"] /* Specify a set of bundled library declaration files that describe the target runtime environment. */,
+        "jsx": "react" /* Specify what JSX code is generated. */,
+        "module": "es2022" /* Specify what module code is generated. */,
```

Comment:
note that this is the first time we're compiling to es modules in this repo. May be relevant later